woocommerce/woocommerce-gateway-payfast
# Update all dependencies

### Description

There are a number of open PRs from dependabot (and some we've already merged in) that update some dependencies in our dependency tree. Instead of handling these one by one, decided to open a single PR that updates everything. We don't include many dependencies ourselves and all the ones we include are `devDependencies`, that are used to compile assets.

This PR replaces:
- #56 
- #57 
- #58 
- #59 
- #74
- #79  
- #81 
- #82 
- #83 

### Steps to Test

1. Checkout this PR
2. If existing install, run `rm -rf node_modules` to start with a clean slate
3. Run `npm install`
4. Ensure any npm scripts work: `npm run build`, `npm run start:webpack`, `npm run lint:js`

Should notice some deprecation warnings when building assets. These should be addressed at some point but they shouldn't cause any issues right now.

In addition, the actual assets we use are for integrating with the WooCommerce Blocks plugin. These were added in #40, so I'd recommend following the testing instructions there. 

### Changelog Entry

> Fix - Update dependencies and fix some code styling.
